ghedesigner/radial_numerical_borehole.py

```diff
@@ -101,34 +99,21 @@ def partial_init(self, single_u_tube: SingleUTube):
         self.t_s = single_u_tube.b.H ** 2 / (9 * soil_diffusivity)
         self.calc_time_in_sec = max([self.t_s * exp(-8.6), 49.0 * SEC_IN_HR])
 
-    def fill_radial_cell(self, radial_cell, resist_p_eq, resist_f_eq, resist_tg_eq):
-
-        num_fluid_cells = self.num_fluid_cells
-        num_conv_cells = self.num_conv_cells
-        num_pipe_cells = self.num_pipe_cells
-        num_grout_cells = self.num_grout_cells
-        num_soil_cells = self.num_soil_cells
+    def fill_radial_cell(self, radial_cell, resist_f_eq, resist_pg_eq):
 
         cell_summation = 0
 
         # load fluid cells
-        for idx in range(cell_summation, num_fluid_cells + cell_summation):
-            center_radius = self.r_fluid + idx * self.thickness_fluid
-
-            if idx == 0:
-                inner_radius = center_radius
-            else:
-                inner_radius = center_radius - self.thickness_fluid / 2.0
-
-            outer_radius = center_radius + self.thickness_fluid / 2.0
+        for idx in range(cell_summation, self.num_fluid_cells + cell_summation):
+            inner_radius = self.r_fluid + idx * self.thickness_fluid_cell
+            center_radius = inner_radius + self.thickness_fluid_cell / 2.0
+            outer_radius = inner_radius + self.thickness_fluid_cell
```

Comment:
The left-most cell was being treated as a half cell. I can't remember why I did it this way to begin with, but I think this should be unnecessary so it is now being treated as a full cell, and does affect the results.